Parse/src/main/java/com/parse/ParseRESTObjectBatchCommand.java

```diff
@@ -63,7 +63,7 @@
       for (ParseRESTObjectCommand command : commands) {
         JSONObject requestParameters = new JSONObject();
         requestParameters.put("method", command.method.toString());
-        requestParameters.put("path", String.format("/1/%s", command.httpPath));
+        requestParameters.put("path", String.format("%s/%s", server.getPath(), command.httpPath));
```

Comment:
@grantland yep, simply via different url logic.

We could solve this by doing `new URL(server, command.httpPath).getPath()`, though that potentially creates a lot of temporary objects in the interim.